roles/common/files/uug_ansible_wrapper.py

```diff
@@ -596,7 +604,6 @@ def validate_branch():
 
     return USER_CONFIG['git_branch'] in ls_remote_output
 
-
```

Comment:
If this is not inside a class, I'd like to keep this empty line. I'd prefer to do:
 - one line between functions inside a class
 - two lines between all global-level functions, between a function and a constant, and between a constant and imports.

As an example of the latter:

```python
import os
import sys


GLOBAL_VAR_A = 1
GLOBAL_VAR_B = 2


class Test(object):
    def __init__(self):
        pass

    def _foo(self):
        pass


def bar():
    pass



def baz():
    pass
```